### PR TITLE
Fix example code to work properly

### PR DESCRIPTION
**Pull request by sjardim**
This PR fixes a couple of small issues to make the example code work. BUT there is still an issue when the project has just one locale (no translations) as when trying to save a page, an exception will be thrown due to the lack of the "setTranslation" method on the Page model (not the TranslatedPage).

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     ],
     "require": {
         "php": "^8.1",
+        "artesaos/seotools": "^1.2",
```
Comment (sten) [inline composer.json:41]: Please, remove. See comment above.

File: example/app/View/Components/BaseLayout.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\View\Components;
+
+use Illuminate\View\Component;
+use Illuminate\View\View;
+
+class baseLayour extends Component
```
Comment (sten) [inline example/app/View/Components/BaseLayout.php:8]: Typo in class name: BaseLayout

**Review by sten — CHANGES_REQUESTED** [review]
@sjardim Thanks a lot for the example upgrade. Can you have a look at my remarks? Thanks!

File: README.md [hunk 3]
```diff
@@ -68,6 +68,7 @@ Here is a brief overview of the choices made:
 - `spatie/laravel-sluggable`: for slugs
 - `spatie/laravel-translatable`: for translations as this works together with the first party filament translatable package. 
 - `dereuromark/media-embed`: to support video embeds of [various media services](https://github.com/dereuromark/media-embed/blob/master/docs/supported.md).
+- `artesaos/seotools`: this is a soft dependency, you can use a different package to handle the SEO fields output if you like.
```
[the remark this answers is not in the document]
Comment (sjardim) [reply README.md:71]: Hi Sten! My pleasure! I'll make the changes and will get back to ya. 

I am also working on another example, integrating [pboivin/filament-peek](https://github.com/pboivin/filament-peek). It's working right now but needs some tweaks. Maybe someone will find it useful :)

Take care!

[the remark this answers is not in the document]
Comment (sten) [reply README.md:71]: @sjardim That would be really cool to integrate it with Filament Peek. We can maybe have a look on how we can closely integrate it in the package if it is not too complicated.

**Pull request closed without merging**